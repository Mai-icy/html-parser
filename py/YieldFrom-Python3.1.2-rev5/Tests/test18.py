#
#   Test exception in initial next() call
#

def g1():
	print("g1 about to yield from g2")
	yield from g2()
	print("g1 should not be here")

def g2():
	yield 1/0

gi = g1()
next(gi)
