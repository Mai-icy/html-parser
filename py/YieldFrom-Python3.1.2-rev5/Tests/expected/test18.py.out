g1 about to yield from g2
Traceback (most recent call last):
  File "test18.py", line 14, in <module>
    next(gi)
  File "test18.py", line 7, in g1
    yield from g2()
  File "test18.py", line 11, in g2
    yield 1/0
ZeroDivisionError: int division or modulo by zero
